Source/WebCore/rendering/RenderGrid.h

```diff
@@ -156,9 +156,16 @@ class RenderGrid final : public RenderBlock {
     GridTrackSizingDirection autoPlacementMajorAxisDirection() const;
     GridTrackSizingDirection autoPlacementMinorAxisDirection() const;
 
-    void allocateSpaceForMasonryVectors(Vector<RenderBox*>& itemsWithDefiniteGridAxisPosition, Vector<RenderBox*>& itemsWithIndefinitePosition); 
-    void collectMasonryItems(Grid&, unsigned gridAxisTracksCount, GridTrackSizingDirection masonryDirection, HashMap<RenderBox*, GridArea>& itemsOnFirstTrack, Vector<RenderBox*>& itemsWithDefiniteGridAxisPosition, Vector<RenderBox*>& itemsWithIndefinitePosition) const;
     bool hasDefiniteGridAxisPosition(const RenderBox* child, GridTrackSizingDirection masonryDirection) const;
+    static bool itemGridAreaStartsAtFirstLine(const GridArea& area, GridTrackSizingDirection masonryDirection)
```

Comment:
Is it possible for these two helpers to be moved as well? They're quite specific to masonry.

Otherwise, appending `InMasonryAxis/Direction` would be good.

The latter I think will be going away once the placement bugs are fixed, so it matters less.

Reply:
Yeah I definitely could! Would these be useful in a non-masonry context in any sort of way? `hasDefiniteGridAxisPosition ` could be changed to `hasDefiniteAxisPosition` so that it could be used to check if an item has a definite span in a specific axis. Otherwise no issues with moving them

Reply:
I'm not sure we have any need for `hasDefiniteAxisPosition` outside of masonry, but either way is fine with me.

Reply:
I'll go ahead and move them unless anyone has any strong opinions otherwise